frontend/src/components/base/Button/Button.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { fireEvent, render } from '../../../test-utils';
+import { Button } from './Button';
+
+describe('Button', () => {
+  it('should render without errors', () => {
+    const { getByRole } = render(<Button type="button">button</Button>);
+    const button = getByRole('button', { name: 'button' });
+
+    expect(button).toBeInTheDocument();
+  });
+
+  it('should produce an event on click if not disabled', () => {
```

Comment:
I like that you're trying to test that the button click event works.  But I'm not sure I understand how exactly it's testing that.  Isn't the `.toBeEnabled` method checking whether or not the button is not disabled

Reply:
Yeah, this is something I intended to address. Adding a mock callback function onClick seems to have done the trick.

Reply:
Ahh yes that would do the trick.  That said, I believe there is a better approach.  Please see my updated comment :)